### DISCUSSION TITLE
Add new configuration files for Real ESRGAN MMagic 新 config 体验与适配 Real ESRGAN

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#654] [MMagic]
# Add new configuration files for Real ESRGAN 
 MMagic 新 config 体验与适配 Real ESRGAN 

According to the [Config adaptation document](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc) and the supported algorithm configurations, add all configurations for Real ESRGAN and verify the correctness of inference and training.

mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配
- [相关适配教程文档](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)
该任务适配的 config 目录：configs/real_esrgan/ ，完成所有文件适配

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：limafang

**Claim Time**：2023年10月25日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by limafang** [comment]
Claim the mission
